app/src/main/res/values/strings.xml

```diff
@@ -61,4 +62,12 @@
 
     <string name="rename_list">Rename List</string>
     <string name="delete_list">Delete List</string>
+
+    <string name="speech_message">Tell us what you want to name your list</string>
```

Comment:
speech_message is too generic, and we want to add speech input to other parts of the app.
```suggestion
    <string name="list_name_input_speech_message">Tell us what you want to name your list</string>
```